src/backends/plotly.jl

```diff
@@ -900,23 +900,49 @@ function plotly_series_segments(series::Series, plotattributes_base::KW, x, y, z
             mcolor = rgba_string(
                 plot_color(get_markercolor(series, clims, i), get_markeralpha(series, i)),
             )
+            mcolor_next =
+                (i + 1 <= length(series[:markercolor])) ?
+                rgba_string(
+                    plot_color(
+                        get_markercolor(series, clims, i + 1),
+                        get_markeralpha(series, i + 1),
+                    ),
+                ) : mcolor
             lcolor = rgba_string(
                 plot_color(
                     get_markerstrokecolor(series, i),
                     get_markerstrokealpha(series, i),
                 ),
             )
+            lcolor_next =
+                (i + 1 <= length(series[:markerstrokecolor])) ?
+                rgba_string(
+                    plot_color(
+                        get_markerstrokecolor(series, i + 1),
+                        get_markerstrokealpha(series, i + 1),
+                    ),
+                ) : lcolor
+
             plotattributes_out[:marker] = KW(
                 :symbol => get_plotly_marker(
                     _cycle(series[:markershape], i),
                     string(_cycle(series[:markershape], i)),
                 ),
                 # :opacity => needs_scatter_fix ? [1, 0] : 1,
                 :size => 2 * _cycle(series[:markersize], i),
-                :color => needs_scatter_fix ? [mcolor, "rgba(0, 0, 0, 0.000)"] : mcolor,
+                :color =>
+                    needs_scatter_fix ?
+                    (
+                        i === length(segments) ? [mcolor, mcolor_next] :
+                        [mcolor, "rgba(0, 0, 0, 0.000)"]
+                    ) : mcolor,
```

Comment:
```suggestion
                :color => needs_scatter_fix ? [mcolor, mcolor_next] : mcolor,
```

Now that we define `mcolor_next`, I think this is what you meant.

Reply:
I was trying to follow the original code as much as possible and added the second marker only to the last segment. I tried your version and the result was the same so I will update as suggested.